docs/hugo/content/contributing/add-a-new-code-generated-resource.md

```diff
@@ -199,5 +199,11 @@ See [the code generator README](../#running-integration-tests) for how to run re
 ## Add a new sample
 The samples are located in the [samples directory](https://github.com/Azure/azure-service-operator/blob/main/v2/config/samples). There should be at least one sample for each kind of supported resource. These currently need to be added manually. It's possible in the future we will automatically generate samples similar to how we automatically generate CRDs and types, but that doesn't happen today.
 
+## Run test for added sample and commit the recording
+The added new sample needs to be tested and recorded. To perform that, follow the steps below:
+1. Check if recording for the `Test_<GROUP>_<VERSION_PREFIX>_CreationAndDeletion.yaml` exists in the [recordings directory](https://github.com/Azure/azure-service-operator/blob/main/v2/internal/controllers/recordings/Test_Samples_CreationAndDeletion) already. For example,  If we're adding sample for NetworkSecurityGroup resource, we need look at `Test_Network_v1beta_CreationAndDeletion.yaml`
```

Comment:
I think you need to be explicit that the name of the file is the same as the name of the test, which only matches your suggested template if they've followed other guidance.

Reply:
Maybe:

```suggestion
1. If a recording for the test already exists, delete it.  
   Look in the [recordings directory](https://github.com/Azure/azure-service-operator/blob/main/v2/internal/controllers/recordings/Test_Samples_CreationAndDeletion) for a file with the same name as your new test.  
   Typically these are named `Test_<GROUP>_<VERSION_PREFIX>_CreationAndDeletion.yaml`.
   For example,  If we're adding sample for NetworkSecurityGroup resource, check for `Test_Network_v1beta_CreationAndDeletion.yaml`
```